src/file.ts

```diff
@@ -2758,6 +2762,30 @@ class File extends ServiceObject<File> {
    * });
    *
    * //-
+   * // Generate a URL that allows temporary access to download your file.
+   * // Access will begin at accessibleAt and end at expires.
+   * //-
+   * const request = require('request');
+   *
+   * const config = {
+   *   action: 'read',
+   *   expires: '03-17-2025',
+   *   accessibleAt: '03-13-2025'
```

Comment:
One potential issue with `accessibleAt` is that it may not fit well with using a Signed URL to upload data only downloading data.

WDYT?

Reply:
From how I read it, 'accessible' refers to the link and not the file. The link to upload is only accessible from the date specified. I think it works.